src/backend/distributed/worker/worker_merge_protocol.c

```diff
@@ -340,7 +339,16 @@ RemoveJobSchema(StringInfo schemaName)
 		 * can suppress notice messages that are typically displayed during
 		 * cascading deletes.
 		 */
-		deleteWhatDependsOn(&schemaObject, showNotices);
+#if (PG_VERSION_NUM >= 100000)
+		performDeletion(&schemaObject, DROP_CASCADE,
+						PERFORM_DELETION_INTERNAL |
+						PERFORM_DELETION_QUIETLY |
+						PERFORM_DELETION_SKIP_ORIGINAL |
+						PERFORM_DELETION_SKIP_EXTENSIONS);
+#else
```

Comment:
No, these flags were added in postgres/postgres@b3427dade14cc31eb48740bc9ea98b5954470b24 so emulating the behavior of `deleteWhatDependsUpon` is not possible in prior PostgreSQL versions.